src/qt/transactiondesc.cpp

```diff
@@ -265,8 +265,8 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx, TransactionReco
         strHTML += "<br><b>" + tr("Comment") + ":</b><br>" + GUIUtil::HtmlEscape(wtx.mapValue["comment"], true) + "<br>";
 
     strHTML += "<b>" + tr("Transaction ID") + ":</b> " + rec->getTxID() + "<br>";
+    strHTML += "<b>" + tr("Transaction total size") + ":</b> " + QString::number(wtx.tx->GetTotalSize()) + " bytes<br>";
```

Comment:
I actually like the way it is ordered now i.e. txid and output index close to each other because together they form a corresponding input. Any reason to re-align with Bitcoin Core besides shrinking the diff?

Reply:
It was really just about fixing a merge conflict I had in a later backport. Changed the order now to what it was before